### ISSUE TITLE
Configurable guard in config

### ISSUE THREAD

**Issue by timo75** [laravel/sanctum#189]
I'm using a custom guard ("admin") for user authentication and looking at the Sanctum's Guard.php file I figured out that it refers to config('sanctum.guard', 'web'), thus using 'web' by default if not set in config.

Is there a reason why something like
'guard' => env('SANCTUM_GUARD', 'web')
hasn't been added to the default config file yet?

**Comment by timo75** [comment]
I know and I certainly did this. My point is that if it was in the published config right away with a little desciption, a newby would quickly realize that the correct guard can be set there. The documentation doesnt mention this at all. One has to dig into the code to figure it out. So basically my intention was to save other people's time ;)

**Comment by driesvints** [comment]
We don't want this to be a much used feature. It's there when people need it but we don't recommend using this.

**Comment by maxacarvalho** [comment]
Hi @driesvints 
I found myself, maybe, in the same situation as @timo75 
I need to use a different guard wit Sanctum SPA authentication, but I got a bit pushed back by your `It's there when people need it but we don't recommend using this.` statement.  

Would you mind to provide more details about why you don't recommend? I mean, it seems pretty common to have more than one guard, one for frontend users and one for admin access.

**Comment by driesvints** [comment]
We're not planning to add this to the config file sorry.

**Comment by maxacarvalho** [comment]
Hello again @driesvints 

Thanks for your quick reply.  
I think I wasn't clear enough, sorry about that, let me try again.

I don't think that adding the setting to the configuration is a problem, I'm happy to browse the code and understand the implementation.  
My question was regarding you statement above, **It's there when people need it but we don't recommend using this**, for me that sounds like a warning, that if someone use it, a problem could occur at some point.  
Is that the case?
